apps/web/src/sitemap.ts

```diff
@@ -0,0 +1,40 @@
+import type { MetadataRoute } from "next";
+
+const addPathToBaseURL = (path: string) => `https://openstatus.dev${path}`;
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: addPathToBaseURL("/"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/play"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-in"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-up"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-in"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-up"),
+      lastModified: new Date(),
+    },
```

Comment:
There is a duplication of paths. Both URL already exists.

Reply:
So I should take them out right?

Reply:
Exactly.